packages/aws-cdk-lib/aws-ecs/lib/base/base-service.ts

```diff
@@ -400,6 +414,14 @@ export interface BaseServiceOptions {
    * @default - undefined
    */
   readonly volumeConfigurations?: ServiceManagedVolume[];
+
+  /**
+   * Indicates whether to use Availability Zone rebalancing for the service
+   *
+   * For more information, see [Amazon ECS Service Rebalancing](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-rebalancing.html)
+   * @default - DISABLED
+   */
+  readonly availabilityZoneRebalancing?: AvailabilityZoneRebalancing;
```

Comment:
I don't think it's correct to put it here. You can't use this feature with EXTERNAL services, and ExternalService extends BaseService.